src/api-requests/get-all-meta-info.ts

```diff
@@ -135,8 +338,95 @@ export const getAllContentTypesMetaInfo = async (
             }
         }
     `;
+
+    const isStrapi = isStrapiClient(client);
+    const query = isStrapi ? strapi_query : cs_query;
+
     const { data }: ApolloQueryResult<{ contentTypes: MetaInfoResponse[] }> =
         await client.query({ query });
 
-    return data.contentTypes;
+    const adaptedData = formatResponse(
+        isStrapi,
+        'contentTypes',
+        data,
+        'ContentType'
+    ) as MetaInfoResponse[];
+
+    return adaptedData;
+};
+
+export const parityTesting = async () => {
```

Comment:
While `parityTesting` is not run in this PR, it could be run and compare the re-formatted ContentStack response is consistent with Strapi's. This helps to check:
1. If the reformatting process is performed correctly
2. If any value is not migrated properly to ContentStack

I don't think this is the right place to have this code, but it was helpful for me debug when I wrote my code.